### PR TITLE
Support deprecated accessing locale directly on Android

### PR DESCRIPTION
**Pull request by ghsdh3409**
## Description

Fixed issue #425

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    NO CHANGE     |
| Android |    ✅     |
| Windows |    NO CHANGE     |

## Checklist

* [x] I have tested this on a device/simulator for each compatible OS
* [x] I added the documentation in `README.md`.
  * No needed
* [x] I mentionned this change in `CHANGELOG.md`.

### CODE REVIEW

File: android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java [hunk 1]
```diff
@@ -67,7 +67,13 @@ private WifiInfo getWifiInfo() {
   }
 
   private String getCurrentLanguage() {
-    Locale current = getReactApplicationContext().getResources().getConfiguration().locale;
+    Locale current;
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+      current = getReactApplicationContext().getResources().getConfiguration().getLocales().get(0);
```
Comment (ghsdh3409) [inline android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: https://developer.android.com/reference/android/content/res/Configuration#locale

**Comment by ohtangza** [comment]
@machour Can you get this merged? Our app uses this without any issue for more than 6 months.

cc: @ghsdh3409

**Comment by ohtangza** [comment]
@machour Thanks! Finally, we ended up making our `package.json` pointing back to the original NPM library. :)

**Comment by machour** [comment]
Seems that this PR did break things for users, could you have a look at #453?

**Comment by ohtangza** [comment]
@machour Our team is checking it now. (Btw, our CI server did not break).

**Pull request merged**